docs/doxygen/ngraph_cpp_api.config

```diff
@@ -0,0 +1,39 @@
+# Doxyfile 1.8.18
+
```

Comment:
why this file is here?

Reply:
added by mistake
will be removed in https://github.com/openvinotoolkit/openvino/pull/9365